README.md

```diff
@@ -49,6 +49,14 @@ QuestDB release and sample datasets:
 - live trading data from a cryptocurrency exchange
 - geolocations of 250k unique ships over time
```

Comment:
We are mentioning the table here, but we are not showing any queries, so not sure why we are mentioning it

Reply:
My reasoning would be - geospatial usage is not our main advantage query-wise. But have this list reflect what we have and perhaps see if we attract users from this sector. Does that make sense?

Reply:
Yes. We also have the weather table and the gas_prices, so it is weird to see here some of the tables. Maybe we can add a demo query with a geohash? With that we could justify having the table listed here, and we would also showcase a real query. Maybe this one? ```select * from pos latest by id where geo6 within(#wtq) and time < '2021-09-19';```

Reply:
> Yes. We also have the weather table and the gas_prices, so it is weird to see here some of the tables. 

`weather` table is only used together with `trips`, and `gas_prices` is not as interesting as the 3 datasets we highlighted, we have only one sample query for `gas_prices`.

IMO we don't need to have queries for all the tables, the queries here are mainly for showcasing common patterns and to include specific clauses like SAMPLE BY, LATEST ON, plus the query performance. If users try to reproduce similar queries they'd be able to see the same level of performance. 

Hence, not much difference between
`select * from pos latest by id where geo6 within(#wtq) and time < '2021-09-19';`
and
`SELECT * FROM trades LATEST ON timestamp PARTITION BY symbol;` 

But then it's better to keep queries at first glance simple. If users seek specific sample queries then can try on demo.